public/models/ffmpegSettings.model.ts

```diff
@@ -0,0 +1,9 @@
+export interface FfmpegSettings {
+  ffmpegPath?: string;
+  readonly audioFileOrFolderPath: string;
```

Comment:
This looks like it is being used for 2 purposes. It might be better to use 2 properties.

Reply:
I don't like it either.  However at this moment the path is being passed down from the UI and would prefer to refactor this "code smell" in a future PR.